### PR TITLE
Switch to branch based model (v1.6.x)

### PR DESCRIPTION
**Pull request by coderbyheart**
See #251

Addresses `v1.6.x` only.

### CODE REVIEW

**Review by wiba-nordic — APPROVED** [review]

File: docs/aws/GettingStarted/FirmwareConfiguration.rst [hunk 1]
```diff
@@ -5,85 +5,61 @@ Configure the firmware
...
 
-   #. Use this MQTT endpoint value as the value for the ``CONFIG_AWS_IOT_BROKER_HOST_NAME`` configuration variable.
+   .. code-block:: bash
+   
+      CONFIG_AWS_FOTA_LOG_LEVEL_DBG=y
+      CONFIG_AWS_IOT_LOG_LEVEL_DBG=y
+      CONFIG_AWS_JOBS_LOG_LEVEL_DBG=y
 
-   #. Use ``42`` as the value for ``CONFIG_AWS_IOT_SEC_TAG``.
+#. Comment out the nRF Connect for Cloud settings in the :file:`prj.conf` file:
```
Comment (peknis) [inline docs/aws/GettingStarted/FirmwareConfiguration.rst:40]: Isn't it nRF Cloud now?

File: docs/aws/GettingStarted/FirmwareConfiguration.rst [hunk 2]
```diff
@@ -5,85 +5,61 @@ Configure the firmware
...
+      # CONFIG_AGPS=y
+      # CONFIG_AGPS_SRC_NRF_CLOUD=y
+      # CONFIG_NRF_CLOUD=y
+      # CONFIG_NRF_CLOUD_AGPS=y
+      # CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD=y
+      # CONFIG_NRF_CLOUD_PGPS=n
+      # CONFIG_NRF_CLOUD_SEND_TIMEOUT=y
+      # CONFIG_GPS_MODULE_NMEA=y
+   
+#. Comment out the nRF Connect for Cloud logging settings in the :file:`overlay-debug.conf` file:
```
Comment (peknis) [inline docs/aws/GettingStarted/FirmwareConfiguration.rst:53]: nRF Cloud?

File: docs/azure/GettingStarted/FirmwareConfiguration.rst [hunk 3]
```diff
@@ -5,91 +5,64 @@ Configure the firmware
...
+      # CONFIG_AGPS=y
+      # CONFIG_AGPS_SRC_NRF_CLOUD=y
+      # CONFIG_NRF_CLOUD=y
+      # CONFIG_NRF_CLOUD_AGPS=y
+      # CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD=y
+      # CONFIG_NRF_CLOUD_PGPS=n
+      # CONFIG_NRF_CLOUD_SEND_TIMEOUT=y
+      # CONFIG_GPS_MODULE_NMEA=y
+   
+#. Comment out the nRF Connect for Cloud logging settings in the :file:`overlay-debug.conf` file:
```
Comment (peknis) [inline docs/azure/GettingStarted/FirmwareConfiguration.rst:56]: nRF Cloud?

**Review by peknis — CHANGES_REQUESTED** [review]

File: docs/aws/GettingStarted/FirmwareConfiguration.rst (hunk 1 above)
Comment (coderbyheart) [reply docs/aws/GettingStarted/FirmwareConfiguration.rst:40]: Yes, but this PR purely focuses on separating the code that is in the v1.6.x branch. We can then backport changes.

**Review by peknis — APPROVED** [review]

**Pull request merged**